### PR TITLE
Add constants for Dynatrace metrics API

### PR DESCRIPTION
**Pull request by pirgeo**
The names of the constants are closest to what I found on the documentation pages, but i am happy to change them if we come up with more descriptive ones.

### CODE REVIEW

File: lib/src/main/java/com/dynatrace/metric/util/DynatraceMetricApiConstants.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dynatrace.metric.util;
+
+public final class DynatraceMetricApiConstants {
+  private DynatraceMetricApiConstants() {}
+
+  private static final String DEFAULT_ONEAGENT_ENDPOINT = "http://127.0.0.1:14499/metrics/ingest";
```
Comment (arminru) [inline lib/src/main/java/com/dynatrace/metric/util/DynatraceMetricApiConstants.java:19]:
```suggestion
  private static final String DEFAULT_ONEAGENT_ENDPOINT = "http://localhost:14499/metrics/ingest";
```
Let's use the same value as we have in the docs.

File: lib/src/main/java/com/dynatrace/metric/util/DynatraceMetricApiConstants.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  private static final int PAYLOAD_LINES_LIMIT = 1000;
+
+  /** @return The default OneAgent endpoint. */
```
Comment (arminru) [inline lib/src/main/java/com/dynatrace/metric/util/DynatraceMetricApiConstants.java:22]:
```suggestion
  /** 
   * @return The default OneAgent endpoint.
   * @see See the <a href="https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/">Dynatrace documentation</a> for more information.
   */
```

File: lib/src/main/java/com/dynatrace/metric/util/DynatraceMetricApiConstants.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  public static String getDefaultOneAgentEndpoint() {
+    return DEFAULT_ONEAGENT_ENDPOINT;
+  }
+
+  /**
+   * @return The maximum number of metric lines per POST request to the /metrics/ingest endpoint.
```
Comment (arminru) [inline lib/src/main/java/com/dynatrace/metric/util/DynatraceMetricApiConstants.java:28]:
```suggestion
   * @return The maximum number of metric lines per request accepted by the ingest endpoint.
```

**Review by arminru — COMMENTED** [review]
Please also mention this in the README.

**Review by dyladan — APPROVED** [review]

**Comment by dyladan** [comment]
failing build but the change looks ok to me other than that :)

**Comment by pirgeo** [comment]
fixed it, seems to have been an issue in generating javadoc.

File: README.md [hunk 4]
```diff
@@ -64,3 +64,20 @@ Since there are multiple levels of dimensions (default, dynamic, OneAgent) and d
...
+
+To access the constants, call the respective static methods on `DynatraceMetricApiConstants`:
+
+```java
+String oneAgentEndpoint = DynatraceMetricApiConstants.getDefaultOneAgentEndpoint();
+```
+
+Currently available constants are:
+
+* the default OneAgent endpoint (`getDefaultOneAgentEndpoint()`)
```
Comment (arminru) [inline README.md:82]:
```suggestion
* the default [local OneAgent metric API](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/) endpoint (`getDefaultOneAgentEndpoint()`)
```

File: README.md [hunk 5]
```diff
@@ -64,3 +64,20 @@ Since there are multiple levels of dimensions (default, dynamic, OneAgent) and d
...
+* the limit for how many lines can be ingested in one request (`getPayloadLinesLimit()`)
```
Comment (arminru) [inline README.md:83]:
```suggestion
* the limit for how many metric lines can be ingested in one request (`getPayloadLinesLimit()`)
```

File: README.md [hunk 6]
```diff
@@ -64,3 +64,20 @@ Since there are multiple levels of dimensions (default, dynamic, OneAgent) and d
 Default dimensions will be overwritten by dynamic dimensions, and all dimensions will be overwritten by OneAgent dimensions if they share the same key after normalization.
 Note that the OneAgent dimensions will only contain [dimension keys reserved by Dynatrace](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/metric-ingestion-protocol/#syntax).
 If the `.withOneAgentMetadata()` method is not called on the `MetricBuilderFactory`, OneAgent metadata will not be queried and added.
+
+### Common constants
+
+The library also provides constants that might be helpful in the projects consuming this library.
+The default OneAgent endpoint is an example: Every exporter using this library will likely provide a way to export to the default OneAgent endpoint.
+Instead of coding the URL into each of these projects, it is provided here in one place.
```
Comment (arminru) [inline README.md:72]: This is rather self-explanatory and could be omitted.
```suggestion
```

**Review by arminru — APPROVED** [review]

**Pull request merged**